scripts/docker/release/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM galaxy-base:latest
+
+RUN useradd -r -m -g root galaxy
+
+# Install tini
+ENV TINI_VERSION v0.16.1
+RUN yum -y install wget \
+    && wget -O /usr/local/bin/tini \
+        "https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini" \
```

Comment:
Would it be a good idea to fork this repo so we don't have to worry about our containers breaking if they change things?